### PR TITLE
[Feature] Upload accounting report to S3

### PR DESCRIPTION
**Pull request by j-bunk**
## Notion ticket link

[Upload accounting report to S3](https://www.notion.so/uwblueprintexecs/Upload-accounting-report-to-S3-c3110beb9bc646b0823d7cf3c414a0af)

## Implementation description
Closely follows https://github.com/uwblueprint/richmond-centre-for-disability/pull/173

## Demo

https://user-images.githubusercontent.com/51224641/165640117-4dfcf532-31f3-4467-9e58-10129f7215a8.mov

[accounting-report-3-20220427-181414.csv](https://github.com/uwblueprint/richmond-centre-for-disability/files/8578113/accounting-report-3-20220427-181414.csv)

image

## Checklist
- [x] My PR name is descriptive, is in imperative tense and starts with one of the following: `[Feature]`,`[Improvement]` or `[Fix]`,
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the RCD team on GitHub, or specific people who are associated with this ticket

### CODE REVIEW

File: lib/reports/resolvers.ts [hunk 1]
```diff
@@ -367,13 +372,37 @@ export const generateAccountantReport: Resolver<
...
+  const csvStringHeader = csvStringifier.getHeaderString();
+  const csvString = csvStringHeader + csvStringRecords;
+
+  // NOTETOSELF: Change name
+  // CSV naming format applications/permit-holders/accounting-report-{employeeID}-{timestamp}.csv
+  const employeeID = session.id;
+  const timestamp = formatDateTimeYYYYMMDDHHMMSS(new Date());
+  // NOTETOSELF: Change fileName and s3InvoiceKey
+  const fileName = `accounting-report-${employeeID}-${timestamp}.csv`;
+  const s3InvoiceKey = `rcd/applications/permit-holderss/${fileName}`;
```
Comment (j-bunk) [inline lib/reports/resolvers.ts:390]: Reused application report name as a placeholder. Any suggestions on what to name `fileName` and `s3InvoiceKey`?

Comment (OustanDing) [reply lib/reports/resolvers.ts:390]: Yes, please name `fileName` as `accounting-report-{employeeID}-{timestamp}.csv` and the key as `rcd/reports/${fileName}`

Comment (OustanDing) [inline lib/reports/resolvers.ts:390]:
```suggestion
  const s3ObjectKey = `rcd/reports/${fileName}`;
```

**Review by OustanDing — CHANGES_REQUESTED** [review]

File: lib/reports/resolvers.ts (hunk 1 above)
Comment (j-bunk) [reply lib/reports/resolvers.ts]: Addressed!

Comment (j-bunk) [reply lib/reports/resolvers.ts]: Addressed!

**Review by OustanDing — APPROVED** [review]
LGTM!

**Pull request merged**